pkg/api/types.go

```diff
@@ -166,6 +166,12 @@ type ObjectMeta struct {
 	// +optional
 	OwnerReferences []metav1.OwnerReference
 
+	// Information regarding the initialization of this object. May be set by privileged users on
```

Comment:
Let me suggest a rewrite to hopefully preemptively reduce the number of questions :)

An initializer is a controller which enforces some system invariant at object creation time. This field is a list of initializers that have not yet acted on this object. If nil or empty, this object has been completely initialized. Otherwise, the object is considered uninitialized and is hidden (in list/watch and get calls) from from clients that haven't explicitly asked to observe uninitialized objects.

When an object is created, the system will populate this list with the current set of initializers. Only privileged users may set or modify this list. Once it is empty, it may not be modified further by any user.